### DISCUSSION TITLE
How can I see the raw request being made?

### DISCUSSION THREAD

**Discussion by ColtonIdle** [usebruno/bruno#1309] [Q&A] [answered]
title ^

**Comment by helloanoop** [comment, marked as answer]
@ColtonIdle  The URL, headers and body that was sent is displayed in the Timeline Tab in the response pane.
Is this what you were looking for ?

![image](https://github.com/usebruno/bruno/assets/9350733/293253c8-9a0d-4c46-b928-ad31a03e8a69)

**Reply by ColtonIdle** [reply]
Yep, that's what I was looking for. I wanted to see how basic auth got transformed into a header. Thanks!

**Reply by andrei-korshikov** [reply]
Not quite so. See #645 and all related issues (#137, #1225, #2037).

**Reply by melroy89** [reply]
Hi. It seem that the timeline is now displaying a beatified body instead of the RAW body:

![image](https://github.com/user-attachments/assets/4dc38486-6da2-4597-9695-c2974dbd7fa9)

So while the headers might look raw, the response is NOT.

I actually needed back to go to curl in order to be able to see the raw content I was looking for. Because I needed to know something about the raw formatting.

**Reply by andrei-korshikov** [reply]
> I actually needed back to go to curl

Yep. I did the same.

**Comment by mattisking** [comment]
I'd like to see more here. I'd really like to see the client certificate attached for one. I may have several certs set and I'd like to see this in the information displayed in the Timeline. It's a feature I frequently use in Postman. I looked into this personally today. It seems like it should be easy until you dig into the code and realize it's not stored in the "item" that gets passed around the various components that ultimately show in the TimeLine. I have plenty of React experience, but not really any Electron. Just messing around there I can debug Bruno just fine, but not the Electron parts and it seems like the certificate used is attached there, not within Bruno. We could fake it by parsing out a match from the collection and displaying that, but that's not really what I'd be looking for. I expect it would be part of the request object that gets passed in to the TimeLine.

**Reply by duyn9uyen** [reply]
There is an open issue on this. Was hoping Bruno v3 fixed this but it didn't. Still no client cert info. https://github.com/usebruno/bruno/issues/3437

**Comment by darkguy2008** [comment]
I think Bruno may be missing a setting to make Web or Raw sticky. When I want to view the raw contents of a response all the time, I have to do that on every request.

Is there a way to make it stick until I change it?
